inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/audit/AuditRequest.java

```diff
@@ -50,9 +50,11 @@ public class AuditRequest {
     @ApiModelProperty(value = "sink id")
     private Integer sinkId;
 
-    @ApiModelProperty(value = "query date, format by 'yyyy-MM-dd'", required = true, example = "2022-01-01")
-    @NotBlank(message = "dt not be blank")
-    private String dt;
+    @ApiModelProperty(value = "query start date, format by 'yyyy-MM-dd'", required = true, example = "2022-01-01")
```

Comment:
Those two params are required? Can we be compatible, and only query the data of the last 1 day, 2 days or 7 days if it is not provided?

Reply:
Those two params are required.